docs/nx-cloud/reference/release-notes.md

```diff
@@ -1,137 +1,139 @@
 # `nx-cloud` - Release notes
 
+## Nx Cloud package
```

Comment:
Why is this file changed?

Reply:
adds a sub heading to this section so it's understood the different parts of the page and what the encompass. was recommended to go this way to make a better structure when solving the multiple h1 tags per page issue from Ben.

Reply:
ref convo: https://nrwl.slack.com/archives/CT3CQ2F0D/p1692041097243949?thread_ts=1692040441.689549&cid=CT3CQ2F0D